sgrep_lint/sgrep.py

```diff
@@ -62,11 +62,24 @@
         help=f"only invoke sgrep if config(s) are valid",
         action="store_true",
     )
+
+    parser.add_argument(
+        "--include",
+        action="append",
+        default=[],
+        help="Scan only files with this filename. Can be added multiple times to include multiple patterns.",
+    )
     parser.add_argument(
         "--exclude",
         action="append",
         default=[],
-        help="Path pattern to exclude. Can be added multiple times to exclude multiple patterns.",
+        help="Don't scan files with this filename. Can be added multiple times to exclude multiple patterns.",
```

Comment:
same comment: is this a filename, or a path pattern? An example might be nice in the helptext too